modlists.json

```diff
@@ -137,5 +137,27 @@
       "NumberOfInstalledFiles": 7905,
       "SizeOfInstalledFiles": 45425318295
     }
+  },
+  {
+    "title": "Sim Settlements Experience",
+    "description": "This is a Modlist designed to showcase the wide variety of add-on packs available for Sim Settlements playthroughs.",
+    "version": "1.2",
+    "author": "SirLach and the Sim Settlements Community",
+    "game": "fallout4",
+    "tags": ["Official"],
+    "links": {
+      "image": "https://raw.githubusercontent.com/wabbajack-tools/mod-lists/master/SSEM/SimSettlementsExperience.png",
+      "readme": "https://raw.githubusercontent.com/wabbajack-tools/mod-lists/master/SSEM/readme.md",
+      "download": "https://drive.google.com/open?id=1QIXZjhXJ8RdySw-W7fjsR4ekwjhSxq0p",
+      "machineURL": "SSEM"
```

Comment:
this should be in all lower case

Reply:
Doh. What do I do now. Do I edit it and resubmit?